site/en/blog/new-in-chrome-104/index.md

```diff
@@ -0,0 +1,210 @@
+I'm [Pete LePage](https://petelepage.com). Let's dive in and
+see what's new for developers in Chrome 104.
+
+## Specify a crop area with region capture {: #region-capture }
+
+`getDisplayMedia` makes it possible to create a video stream from the current
+tab. But, there are times when you don’t want the entire tab, only a small
+portion of it. Until now, the only way to do that was to manually crop each
+video frame.
+
+With Region Capture, a web app can define the specific area of the screen it
+wants to share. For example, Google Slides could allow you to stay in the
+standard editing view, and share the current slide.
+
+<figure>
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/L87rANm3IfsVZiMHfbH2.png", alt="Screenshot of a browser window featuring a web app highlighting the main content area and cross-origin iframe.", width="800", height="568" %}
+  <figcaption>
+    The main content area is in blue and the cross-origin iframe is in red.
+  </figcaption>
+</figure>
+
+To use it, select the element to share, then create a new `CropTarget` based on
+that element. Next, start screen sharing by calling `getDisplayMedia()`, that
+prompts the user for permission to share their screen. Then, call
+`track.cropTo()` and pass the `cropTarget` created earlier.
+
+```js
+const elem = document.querySelector("#main");
+const cropTarget = await CropTarget.fromElement(elem);
+
+const stream = await navigator.mediaDevices
+                    .getDisplayMedia({preferCurrentTab: true});
+const [track] = stream.getVideoTracks();
+
+await track.cropTo(cropTarget);
+```
+
+Check out [Better tab sharing with Region Capture](/docs/web-platform/region-capture/)
+for more details.
+
+## Easier media queries with level 4 syntax and evaulation {: #mq-math }
+
+Media Queries are critical to responsive design, allowing you to define
+specific styles for different viewport sizes. But, unless you use them every
+single day, the syntax can be a little confusing.
+
+Chrome 104 adds support for [Media Queries - Level 4 - Syntax and Evaluation][mq-l4-se],
+allowing you to write media queries using ordinary mathematical comparison
+operators.
+
+So instead of something like this to indicate a viewport between 400 and 600
+pixels:
+
+```css
+@media (min-width: 400px) and (max-width: 600px) {
+  /* Styles for viewports between 400px and 600px. */
+}
+```
+
+It can be written like this:
+
+```css
+@media (400px <= width <= 600px) {
+  /* Styles for viewports between 400px and 600px. */
+}
+```
+
+In addition to making media queries less verbose, the new syntax can be more
+accurate. The `min-` and `max-` queries are inclusive, for example:
+`min-width: 400px` tests for a width of 400px or greater. The new syntax
+allows you to be more explicit about what you mean.
+
+```css
+@media (width < 400px) {
+  /* Styles for viewports less than 400px. */
+}
+@media (400px <= width <= 600px) {
+  /* Styles for viewports between 400px and 600px. */
+}
+@media (601px <= width <= 1000px) {
```

Comment:
Is this on screen in the video? My first though was , why didn't he change the inclusivity on the third example? Then, "Oh, duh." I wonder if this jumps out better:

`@media (600px < width <= 1000px)`

Reply:
Yes, it's on screen.